### PR TITLE
[Go] Create exercises folder with basic READMEs

### PR DESCRIPTION
**Pull request by tehsphinx**
This is mostly a copy of C#'s folder with some adjustments to Go.

### CODE REVIEW

File: languages/go/exercises/.docs/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Go exercise documents
```
Comment (ErikSchierboom) [inline languages/go/exercises/.docs/README.md]: The location for this file should change to `exercises/shared/.docs/README.md`. See #54.

File: languages/go/exercises/.docs/cli.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# CLI (Command Line Interface)
```
Comment (ErikSchierboom) [inline languages/go/exercises/.docs/cli.md]: The location for this file should change to `exercises/shared/.docs/cli.md`. See #54.

File: languages/go/exercises/.docs/debug.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Debug
```
Comment (ErikSchierboom) [inline languages/go/exercises/.docs/debug.md]: The location for this file should change to `exercises/shared/.docs/debug.md`. See #54.

File: languages/go/exercises/.docs/cli.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+You can run the tests by opening a command prompt in the exercise's directory, and then running the [`go test` command][docs-go-test].
+Alternatively, most IDE's have built-in support for running tests, including [Visual Studio Code][docs-run-unit-tests-visual-studio-code] and [JetBrains GoLand][docs-run-unit-tests-goland].
+
+Initially, only the first test will be enabled. This is to encourage you to solve the exercise one step at a time. Once you get the first test passing, remove the `Skip` property from the next test and work on getting that test passing.
+
+Once none of the tests are skipped and they are all passing, you can submit your solution by opening a command prompt in the exercise's directory and running the
```
Comment (ErikSchierboom) [inline languages/go/exercises/.docs/cli.md]: There is an unneeded newline between "running the" and "using."

File: languages/go/exercises/concept/README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+It's only important that it's reasonably easy to _find_ the exercise. It's okay if the name isn't perfect. We **will** iterate on this.
+
+## Contributing
+
+Thanks for wanting to contribute to the Go track's concept exercises! Contributions are very welcome!
+
+To contribute, please find and work on one of the [new exercise issues][issues-new-exercise] or [improve exercise issues][issues-improve-exercise].
+
+[docs-concept-exercises]: ../../reference/README.md
+[issues-new-exercise]: https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Atrack%2Fcsharp+label%3Atype%2Fnew-exercise+label%3Astatus%2Fhelp-wanted
```
Comment (ErikSchierboom) [inline languages/go/exercises/concept/README.md]:
```suggestion
[issues-new-exercise]: https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Atrack%2Fgo+label%3Atype%2Fnew-exercise+label%3Astatus%2Fhelp-wanted
```

File: languages/go/exercises/concept/README.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+[issues-improve-exercise]: https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Atrack%2Fcsharp+label%3Atype%2Fimprove-exercise+label%3Astatus%2Fhelp-wanted
```
Comment (ErikSchierboom) [inline languages/go/exercises/concept/README.md]:
```suggestion
[issues-improve-exercise]: https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Atrack%2Fgo+label%3Atype%2Fimprove-exercise+label%3Astatus%2Fhelp-wanted
```

**Comment by tehsphinx** [comment]
@ErikSchierboom:  Thx! changes applied.

**Pull request merged**